### DISCUSSION TITLE
Gloabl Adjustment value

### DISCUSSION THREAD

**Discussion by vijayank88** [The-OpenROAD-Project/OpenROAD-flow-scripts#840] [General] [answered]
@maliberty 
In global routing log i found following:
`[INFO GRT-0022] Global adjustment: 0%`

Is above value taken from [here](https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/blob/80750431b52961c35d97b907dd830b4351cf9ba5/flow/scripts/global_route.tcl#L12) or where it's captured?

**Comment by maliberty** [comment, marked as answer]
That is only used when set_global_routing_layer_adjustment is used with '*' for the layers.
